obspy/clients/iris/tests/test_client.py

```diff
@@ -234,14 +234,15 @@ def test_resp(self):
         t2 = UTCDateTime("2008-001T00:00:00")
         result = client.resp("IU", "ANMO", "00", "BHZ", t1, t2)
         self.assertIn(b'B050F03     Station:     ANMO', result)
+        # Exception: No response data available
```

Comment:
Maybe choose a different response so that we can still test this part of the code?